run

```diff
@@ -27,7 +27,7 @@ $RUN --service-ports mermaid sh -c "npx pnpm run dev"
 ;;
 
 docs:dev)
-$RUN --service-ports mermaid sh -c "cd packages/mermaid/src/docs && npx pnpm prefetch && npx vitepress --port 3333 --host"
+$RUN --service-ports mermaid sh -c "npx pnpm run --filter mermaid docs:dev:docker"
```

Comment:
Would it make more sense to put Docker specific scripts here instead of in the `package.json` files?

Like with something like:

```suggestion
$RUN --service-ports mermaid sh -c "npx pnpm run --filter mermaid docs:pre:vitepress && npx concurrently 'pnpm --filter docs exec vitepress --port 3333' 'pnpm --filter mermaid exec ts-node-esm src/docs.mts --watch --vitepress'"
```

**Pros:** Simplifies `package.json` file and hides the scripts from most people.
**Cons:** If somebody changes one of these scripts, they might forget to update this script.

Reply:
It used to be there, but my motivation is to keep it as obvious as possible for developers who use classic (no docker) setup. Its highly likely that they will forget to update `run` bash script along with these `package.json` scripts they are familiar with.

Reply:
:+1: In that case, I approve!